### ISSUE TITLE
ng add @angular/fire -> package install fails

### ISSUE THREAD

**Issue by ryanlogsdon** [angular/angularfire#3321]
### Version info

**Angular: 15.2.1**

**Firebase: 9.17.2**

**AngularFire: 7.5.0**

**Other (e.g. Ionic/Cordova, Node, browser, operating system): Node 18.14.2**

### How to reproduce these conditions

**Failing test unit, Stackblitz demonstrating the problem**

from the terminal, `ng add @angular/fire` fails.

it correctly asks _what features would you like_, then _which login?_, but when it asks you to _please select a project_, the command breaks.

To fix this bug, you need to first run:

`ng generate environments`
then
`npm install @angular/fire`
then
`ng add @angular/fire` 

**Steps to set up and reproduce**

**Sample data and security rules**

### Debug output

** Errors in the JavaScript console **

** Output from `firebase.database().enableLogging(true);` **

** Screenshots **

**Comment by mdesoto** [comment]
This just happened to me. Some additional context:

```
 % ng add @angular/fire
ℹ Using package manager: npm
✔ Found compatible package version: @angular/fire@0.0.0.
✔ Package information loaded.

The package @angular/fire@0.0.0 will be installed and executed.
Would you like to proceed? Yes
✔ Packages successfully installed.
Package "@angular/fire" was found but does not support schematics.
```
So, successful but unsuccessful.

The short-term remedy is to use `npm` to remove and then to add `@angular/fire`:

```
% npm remove @angular/fire
% npm install @angular/fire
```

**Comment by jamesdaniels** [comment]
Fixed in 7.6.0, which should be released momentarily
